internal/testutil/acc/cluster.go

```diff
@@ -27,14 +28,11 @@ type ClusterInfo struct {
 // GetClusterInfo is used to obtain a project and cluster configuration resource.
 // When `MONGODB_ATLAS_CLUSTER_NAME` and `MONGODB_ATLAS_PROJECT_ID` are defined, creation of resources is avoided. This is useful for local execution but not intended for CI executions.
 // Clusters will be created in project ProjectIDExecution.
-func GetClusterInfo(tb testing.TB, req *ClusterRequest) ClusterInfo {
+func GetClusterInfo(tb testing.TB, req *ClusterRequest, specs ...admin.ReplicationSpec) ClusterInfo {
```

Comment:
i see the convenience of having specs... , but not sure if i'd have it in ClusterRequest as all other params

Reply:
It makes the calls prettier IMO. 
Otherwise, we would have to pass a `[]admin.ReplicationSpec{}` to the ClusterRequest.
That would be uglier IMO